src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,47 @@
 import React from 'react';
+import { CommentList } from '../CommentList';
+import { Post } from '../types/Post';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          {user && <UserInfo user={user} />}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      {comments.length === 0
+        ? (
+          <>
+            <hr />
+
+            <b data-cy="NoCommentsMessage">No comments yet</b>
+          </>
+        )
+        : <CommentList comments={comments} /> }
+    </div>
```

Comment:
```suggestion
      {comments.length
        ? (
          <>
            <hr />

            <b data-cy="NoCommentsMessage">No comments yet</b>
          </>
        )
        : <CommentList comments={comments} /> }
    </div>
```

please inverse the condition